src/main/java/seedu/address/logic/commands/GradebookFindCommand.java

```diff
@@ -31,8 +31,8 @@ public GradebookFindCommand (Gradebook gradebookComponent) {
     @Override
     public CommandResult execute(Model model, CommandHistory history) {
         requireNonNull(model);
-        CommandResult result = GradebookManager.findGradebookComponent(toFindGradebookComponent.getModuleCode(),
+        String result = GradebookManager.findGradebookComponent(toFindGradebookComponent.getModuleCode(),
                 toFindGradebookComponent.getGradeComponentName());
```

Comment:
This implementation needs to change in future because it is rather restrictive in usage as it stands